### ISSUE TITLE
Elements within the sidebar are accessible with the keyboard while hidden

### ISSUE THREAD

**Issue by LeBenLeBen** [frctl/fractal#830]
While reviewing #775 I realized that the element were never really hidden but just put outside of the viewport. This is an issue as the navigation element can be reached with the keyboard.

The wrapper should have a `display: none` or equivalent property so these cannot be reached.
